chain/x/oracle/client/cli/tx.go

```diff
@@ -48,6 +48,7 @@ func GetTxCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
 		GetCmdActivate(cdc),
 		GetCmdAddReporter(cdc),
 		GetCmdRemoveReporter(cdc),
+		GetCmdAddReporters(cdc),
```

Comment:
I don't think we need a new command. Let's change the current `add-reporter` to take any number of reporters.